_posts/2021-03-10-Bootable-jar-jkube-clustering-dns-ping-openshift.adoc

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+date:   2021-03-10
```

Comment:
Can you change this date and the file name to 2021-03-12 please?

The WF 23 release announcement will go out tomorrow and I want it to be the first post on the list that day. But I don't want this getting buried under it either; it's really good and I want it on top. Merging it Friday will achieve that.